core/src/stmt.rs

```diff
@@ -25,7 +25,7 @@ pub trait MongoStatement: Debug {
     }
     // Executes a prepared statement.
     // Only MongoQuery supports this workflow. The other statements don't.
-    fn execute(&mut self, _: &MongoConnection, _: Bson) -> Result<bool> {
+    fn execute(&mut self, _: &MongoConnection, _: Bson, _: u32) -> Result<bool> {
```

Comment:
Can we do this to properly keep track of what the arguments are, even here?
```suggestion
    fn execute(&mut self, _connection: &MongoConnection, _stmt_id: Bson, _rowset_size: u32) -> Result<bool> {
```